README.md

```diff
@@ -414,18 +414,32 @@ services:
 | Parameter                                | Description                                                                   | Default        |
 |:---------------------------------------- |:----------------------------------------------------------------------------- |:-------------- |
 | `alerting.pagerduty`                     | Configuration for alerts of type `pagerduty`                                  | `{}`           |
-| `alerting.pagerduty.integration-key`     | PagerDuty Events API v2 integration key.                                      | Required `""`  |
+| `alerting.pagerduty.integration-key`     | PagerDuty Events API v2 integration key.                                      | `""`           |
 | `alerting.pagerduty.default-alert`       | Default alert configuration. <br />See [Setting a default alert](#setting-a-default-alert) | N/A       |
+| `alerting.pagerduty.integrations`        | Pagerduty integrations per team configurations                                |                |
+| `alerting.pagerduty.integrations`        | Pagerduty integrations per team configurations                                |   `[]`         |
```

Comment:
```suggestion
```
Could you remove these two duplicate lines?

Reply:
Sure